### DISCUSSION TITLE
TRACE-MultiJunction component implementation

### DISCUSSION THREAD

**Discussion by NRCgg** [NRC-Research/SNAP-issues#59] [General]
@WDunsford && @dbarber-islinc : 

Is now a good time to take a look at implementing the [TRACE-MultiJunction](https://github.com/NRC-Research/SNAP-TRACE-zipper-design) component?  I am thinking that the design may have to be updated considering work that has been done with the Python bindings.  

Thoughts on this?

**Comment by WDunsford** [comment]
I think now's an excellent time for this.  We're about to release TRACE plug-in version 4.6.1, so this would go into 4.6.2 (or later, depending on timing).  We'll include Python bindings and Diff/Merge with the initial cut. We're also going to start work on the "Advanced Python Bindings" which allows for adding / removing components, and resizing tabular components.
